qualification/__init__.py

```diff
@@ -48,10 +48,12 @@
 from spead2.recv.numba import chunk_place_data
 
 import katgpucbf.recv
-from katgpucbf import COMPLEX
+from katgpucbf import COMPLEX, DIG_SAMPLE_BITS
 from katgpucbf.spead import DEFAULT_PORT, FREQUENCY_ID, TIMESTAMP_ID
 from katgpucbf.utils import TimeConverter
 
+from .reporter import Reporter as _Reporter
```

Comment:
I should've also asked, why underscore-Reporter?

Reply:
I didn't want code to start writing `from . import Reporter` in place of `from .reporter import Reporter` i.e. I didn't want it to become part of the public API of this file. It's not really consistent though because e.g. TimeConverter ends up being part of the public API. I guess the difference is that TimeConverter is not something you'd likely assume was part of the public API (because it's defined in a completely different place), whereas Reporter is part of the qualification tests.

I'm not committed to it. @james-smith-za do you have an opinion?

Reply:
> I guess the difference is that TimeConverter is not something you'd likely assume was part of the public API (because it's defined in a completely different place), whereas Reporter is part of the qualification tests.

My thoughts exactly. With this overall reasoning (especially `from .reporter` instead of `from . import`), I'm happy for it to stay as `_Reporter` (underscore-Reporter). However, 
* does that mean we need to update other usages under `qualification/` (or at this level in the dir)? Or
* Can everything stay the way it is, and we'll use this going forward?

Reply:
I have mixed feelings about this.

- I don't think it's particularly important for now. There is no one really using this repo except us, so "Public API" isn't something that we need to be overly concerned about.
- That being said though, there is a TODO comment from yours truly at the top of the file which talks about perhaps there's enough in this file that it warrants its own module.

It looks like the stuff declared in this file gets imported in the qualification tests here:
https://github.com/ska-sa/katgpucbf/blob/f2da9667cf3e46b5a63f1d85cf810c8b26b55f56/qualification/conftest.py#L41
So it may be worth a technical debt ticket to move the bulk of this code into their own modules, and then have the `__init__.py` mostly blank and explicitly controlling what we want to be in the public API which I think will likely be very little. It should probably just exist so that we can do relative imports (though I'm a bit hazy on how that works, it's been a while).

Reply:
True, "public" users are really just the individual qualification tests, rather than code in other git repositories, so we're in control of the usage. I'll change it to get rid of the underscore.